client/src/views/home/HomePage.vue

```diff
@@ -294,6 +294,26 @@ async function handleQuery(): Promise<void> {
   queryInProgress.value = false;
 }
 
+async function onJoinOrganizationClicked(): Promise<void> {
+  const link = await getTextFromUser({
```

Comment:
`getTextFromUser` modal is too big for its content

![Capture d’écran 2025-04-25 à 12 36 03](https://github.com/user-attachments/assets/62270a9b-0627-487b-b6d8-4dde4e2912c6)